src/custom-types.json

```diff
@@ -631,6 +649,18 @@
       "http://stackoverflow.com/a/12770116"
     ]
   },
+  "image/x-nikon-nef" : {
+    "extensions": ["nef"],
+    "sources": [
+      "https://stackoverflow.com/a/47612661/816478"
```

Comment:
The sources needs to contain a [primary source](https://en.wikipedia.org/wiki/Primary_source) for the type data (https://github.com/jshttp/mime-db#contributing).